src/Butler/App/TodoManager.hs

```diff
@@ -276,12 +359,10 @@ unSetEditingTask todoManager =
 startTodoManager :: AppContext -> ProcessIO ()
 startTodoManager ctx = do
     logInfo "TodoManager started!" []
-    let appState = TodoManager 0 NoEditingTask []
+    let appState = TodoManager 0 NoEditingTask [] (TodoSettings True True) False
```

Comment:
nit: For such default value, it's good to have a `newTodoManager :: TodoManager` helper defined next to the data type declaration.